neo4j-to-neptune/src/main/java/com/amazonaws/services/neptune/ConvertCsv.java

```diff
@@ -66,10 +88,36 @@ public void run() {
 
             Directories directories = Directories.createFor(outputDirectory);
 
+            File input = inputFile;
+            File tempDataFile = null;
+
+            // if no input file provided, it is via streaming
+            if (input == null) {
+                String uriInput, usernameInput, passwordInput;
+                if (envFile != null) {
```

Comment:
It might be nice to attempt to merge the dotenv and cli args together instead of using all dotenv or all cli args. For example I might want to store my URI and username in the file, but pass the password via CLI. I think if the arguments are specified in 2 locations, the CLI argument should override the dotenv arg.